.github/workflows/pr_helper.yml

```diff
@@ -56,15 +56,3 @@ jobs:
               repo: context.repo.repo,
               body: '${{env.MESSAGE}}'
             })
-
```

Comment:
not sure we should get rid of it actually. let's leave for now.

Reply:
+1. Was there some context for why we wanted to remove this?